hack/release/snapshot.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+GIT_TAG=$(git describe --exact-match --tags || echo "no tag")
+HEAD_HASH=$(git rev-parse HEAD)
+
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+source "${SCRIPT_DIR}/common.sh"
+config
+
+# Create a snapshot release
+snapshot "$HEAD_HASH"
+
+git reset --hard
```

Comment:
I don't think we need these calls anymore if we aren't doing anything after this. Can just finish this script after the call to the snapshot function

Reply:
Wouldn't the 'dirty' state persist after this script ends? Since release is being called immediately after I didn't think we could remove it.

Reply:
Since we've separated into separate workflows, state won't be shared. Resolving.